homeassistant/components/flux_led/config_flow.py

```diff
@@ -299,7 +299,8 @@ async def _async_try_connect(
             # AKA `HF-LPB100-ZJ200`
             return device
         bulb = async_wifi_bulb_for_host(host, discovery=device)
-        bulb.discovery = discovery
+        if discovery is not None:
+            bulb.discovery = discovery
```

Comment:
Is this related to the dependency bump?

Reply:
The upstream library expects a `not None` object for the `discovery` setter. _Even though it internally handles `None` as well._

Opened https://github.com/Danielhiversen/flux_led/pull/405 upstream to correct that.